### ISSUE TITLE
Acquire Zoom account

### ISSUE THREAD

**Issue by froboy** [mosa-chicago/mosa#23]
It'd be good to have a pro Zoom account for holding meetings. We've currently been piggybacking on a few folks, but we should get our own. @mlhess has offered to grant us a pro account, but we need to discuss how that will work.

**Comment by kthull** [comment]
Alternatively, Techsoup will do at $75/year sub (50% off) after paying a $65 admin fee to access the discount.

**Comment by froboy** [comment]
Here's the [TechSoup offer](https://www.techsoup.org/products/zoom-meetings-pro-plan-bundle--1-year-subscription-%E2%80%93-access-to-discounted-rates--G-50363--).

@mlhess has generously granted us a license through a bundle he has for work. He's ensured me that this will stay active for a relatively indefinite period (as long as... he's teaching at UMich and Zoom is the preferred way for meeting). I think that's good enough for now.

I've set up a group email and invited the `mosa-board` group to it. I've created and linked the Zoom account and the creds are in the MOSA Board 1Password vault. Anyone who'd like to schedule meetings with it is welcome.
